### PR TITLE
itervalues and iteritems replaced by six for python3 compatibility

### PR DESCRIPTION
**Pull request by davidlange6**
partial implementation of requested changes in #23451 
(still to do keys() and values())

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
please test

File: SimG4CMS/HGCalTestBeam/test/HGCalTBGenSimDigiReco_cfg.py [hunk 1]
```diff
@@ -110,7 +111,7 @@
 for path in process.paths:
         getattr(process,path)._seq = process.generator * getattr(process,path)._seq
 
-for label, prod in process.producers_().iteritems():
+for label, prod in process.producers_(six.iteritems()):
```
Comment (kpedro88) [inline SimG4CMS/HGCalTestBeam/test/HGCalTBGenSimDigiReco_cfg.py]: I don't think this one is right... similar issue in the next two files below, and several others (search for `six.iteritems()`)

Comment (davidlange6) [reply SimG4CMS/HGCalTestBeam/test/HGCalTBGenSimDigiReco_cfg.py]: indeed, that is wrong.  my script didn't handle it correctly. thanks for noticing

**Comment by davidlange6** [comment]
will fix with a new pr

**Pull request closed without merging**